R/mod_mappingTab.R

```diff
@@ -31,8 +31,10 @@ mappingTabUI <- function(id, meta, domainData, mappings=NULL, standards=NULL){
   metaDomains <- unique(meta$domain)
   dataDomains <- names(domainData)
   domains <- metaDomains[metaDomains %in% dataDomains]
-
+  print(standards)
```

Comment:
Same comment here. Should this be removed?

Reply:
```suggestion

```